src/components/Inputs/index.tsx

```diff
@@ -20,11 +20,12 @@ import { motion, Variants } from 'framer-motion'
 import styled, { css } from 'styled-components'
 import tinycolor from 'tinycolor2'
 
-export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
-  error?: string
+export interface InputProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'placeholder'> {
```

Comment:
Mmm, `placeholder` not already optional on `HTMLInputElement`? Interesting they did that.

Reply:
I need to override it because it expects `string | undefined` and it conflicts with the `null` or `ReactNode`.